### PR TITLE
MAISTRA-1208: Use a file for oauth secret cookies

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

**Comment by jwendell** [comment]
I tested it in a OCP/OSSM 1.1 deployment and it worked fine, as expected.

File: pkg/controller/servicemesh/controlplane/htpasswd.go [hunk 1]
```diff
@@ -95,6 +95,34 @@ func (r *controlPlaneInstanceReconciler) patchGrafanaConfig(ctx context.Context,
...
 }
 
+func (r *controlPlaneInstanceReconciler) patchProxySecret(ctx context.Context, object *unstructured.Unstructured) error {
+	var rawPassword string
+	log := common.LogFromContext(ctx)
+
+	proxySecret := &corev1.Secret{}
+	err := r.Client.Get(ctx, client.ObjectKey{Namespace: object.GetNamespace(), Name: object.GetName()}, proxySecret)
+	if err == nil {
+		rawPassword = string(proxySecret.Data["session_secret"])
```
Comment (rcernich) [inline pkg/controller/servicemesh/controlplane/htpasswd.go:105]: Is this already base64 encoded?

Comment (jwendell) [reply pkg/controller/servicemesh/controlplane/htpasswd.go:105]: `Get()` gives it to us as plain text.

**Review by rcernich — APPROVED** [review]

**Review by dgn — APPROVED** [review]

**Pull request merged**